Dockerfile.ginkgo

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
yes -- actually, need to enable a mode to not require dockerfiles for some of the executables